### ISSUE TITLE
ordered_with_getter_and_setter_first dont ignore protected functions

### ISSUE THREAD

**Issue by prisis** [PedroTroller/PhpCSFixer-Custom-Fixers#47]
Hey :),

#### The command you use to run PHP CS Fixer:
=> php vendor/bin/php-cs-fixer fix

#### The configuration file you are using, if any:
```php
<?php

$config = PhpCsFixer\Config::create()
    // ...
    ->setRules([
        // ...
        'PedroTroller/ordered_with_getter_and_setter_first' => true,
        'ordered_class_elements' => true,
    ])
    // ...
    ->registerCustomFixers(new PedroTroller\CS\Fixer\Fixers())
;

return $config;
```

#### If applicable, please provide minimum samples of PHP code (as plain text, not screenshots):
 * before running PHP CS Fixer (no changes):
```php
<?php

class Test {
	public function getDefaultOptions(): array {
	}

	protected function getOptions(): array {
	}
}
```

 * with unexpected changes applied when running PHP CS Fixer:
```php
<?php

class Test {
	protected function getOptions(): array {
	}

	public function getDefaultOptions(): array {
	}
}
```

 * with the changes you expected instead:
```php
<?php

class Test {
	public function getDefaultOptions(): array {
	}

	protected function getOptions(): array {
	}
}
```

I expected that ordered_with_getter_and_setter_first is run before ordered_class_elements fixer and the protected functions are not moved above the public one

**Comment by PedroTroller** [comment]
:+1: let me check.

**Comment by PedroTroller** [comment]
@prisis fixed with v2.15.2.
Thanks ;)

**Comment by prisis** [comment]
Thank you!
